### ISSUE TITLE
Arch Linux, psycopg2 (2.7.1) error

### ISSUE THREAD

**Issue by grzekru** [odoo/odoo#19511]
Impacted versions: 10.0

Steps to reproduce:
OS: Arch Linux
Psycopg2 version: 2.7.1 (works good with 2.7.3.1)
Postgresql: 9.6.5
Error occurs when starting odoo

Current behavior:
```
Traceback (most recent call last):
  File "./odoo-bin", line 5, in <module>
    __import__('pkg_resources').declare_namespace('odoo.addons')
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2175, in declare_namespace
    declare_namespace(parent)
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2191, in declare_namespace
    _handle_ns(packageName, path_item)
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2126, in _handle_ns
    loader.load_module(packageName)
  File "/usr/lib/python2.7/pkgutil.py", line 246, in load_module
    mod = imp.load_module(fullname, self.file, self.filename, self.etc)
  File "/home/gk/Software/odoo10/odoo/__init__.py", line 60, in <module>
    import modules
  File "/home/gk/Software/odoo10/odoo/modules/__init__.py", line 8, in <module>
    from . import db, graph, loading, migration, module, registry
  File "/home/gk/Software/odoo10/odoo/modules/graph.py", line 13, in <module>
    import odoo.osv as osv
  File "/home/gk/Software/odoo10/odoo/osv/__init__.py", line 4, in <module>
    import osv
  File "/home/gk/Software/odoo10/odoo/osv/osv.py", line 4, in <module>
    from ..exceptions import except_orm
  File "/home/gk/Software/odoo10/odoo/exceptions.py", line 15, in <module>
    from tools.func import frame_codeinfo
  File "/home/gk/Software/odoo10/odoo/tools/__init__.py", line 9, in <module>
    from convert import *
  File "/home/gk/Software/odoo10/odoo/tools/convert.py", line 23, in <module>
    from .translate import _
  File "/home/gk/Software/odoo10/odoo/tools/translate.py", line 27, in <module>
    from odoo import sql_db, SUPERUSER_ID
  File "/home/gk/Software/odoo10/odoo/sql_db.py", line 18, in <module>
    import psycopg2
  File "/usr/lib/python2.7/site-packages/psycopg2/__init__.py", line 50, in <module>
    from psycopg2._psycopg import (                     # noqa
ImportError: /usr/lib/python2.7/site-packages/psycopg2/.libs/libresolv-2-c4c53def.5.so: symbol __res_maybe_init, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
```
Expected behavior: Odoo starting normally.

Video/Screenshot link (optional):

**Comment by categulario** [comment]
As mentioned here https://github.com/psycopg/psycopg2-wheels/issues/2, try installing the no-binary version of psycopg2

**Comment by categulario** [comment]
As mentioned here https://github.com/psycopg/psycopg2-wheels/issues/2, try installing the no-binary version of psycopg2

**Comment by shahinism** [comment]
Upgrading to the latest version of `psycopg2` (2.7.3.1) solved the issue for me.

**Comment by Yenthe666** [comment]
@d-fence maybe something else for you to check while you're on dependencies and installers?

**Comment by tafaRU** [comment]
FYI, just faced the same error in my own machine:
```
ImportError: /home/tafaru/dev/envs/odoo-dev/odoo-11.0/lib/python3.6/site-packages/psycopg2/.libs/libresolv-2-c4c53def.5.so: symbol __res_maybe_init, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
```
and I fixed with:

`odoo-11.0/bin/pip3 install psycopg2==2.7.3.1` as highlighted at https://github.com/odoo/odoo/issues/19511#issuecomment-336065874.

**Comment by mart-e** [comment]
Thanks for the report, fixed at e419f395a123040896dbe942ad7f0ce10983d197
